src/ExternalModule.ts

```diff
@@ -58,7 +58,8 @@ export default class ExternalModule {
 			},
 			isEntry: false,
 			isExternal: true,
-			meta
+			meta,
+			syntheticNamedExports: false
```

Comment:
I wonder if we can somehow allow to use `syntheticNamedExports` with external modules, it will be great!

Reply:
It would not work the same way as for internal modules. For internal modules, syntheticNamedExports are only used for named exports that cannot be resolved. For external modules, we do not know which exports can or cannot be resolved. So the question is how to go about this. Either just make all named exports "synthetic" or implement a runtime detection logic. And how would this interact with the runtime detection logic for `__esModule`? If you want to get missing named exports from the default export, should it be the exported namespace or the `namespace.default` property? (In real life use-cases, likely the former).

And for formats other than ESM, you can already import whatever you want from external dependencies without errors being thrown.

In short, these are the reasons why we do not have this at the moment.